api/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ApiConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'api'
```

Comment:
I'm not sure about this way of adding API to the application. We usually keep the api views and serializers in our django apps, for example:

```
teaching_platform/
├─ projects/
│  ├─ admin.py
│  ├─ api_views.py
│  ├─ models.py
│  ├─ serializers.py
│  ├─ views.py
├─ schedules/
│  ├─ admin.py
│  ├─ api_views.py
│  ├─ models.py
│  ├─ serializers.py
│  ├─ views.py
```

with your current setup you can easily get to the point where your `serializers.py` or `views.py` have 1000+ lines of code. And it's something that we usually want to avoid :slightly_smiling_face: